### PR TITLE
Fast forward to latest script distributed from official site by today

### PR DESCRIPTION
**Pull request by Enelar**
While i was on patching this script i get noticed that repository is out of sync. It is GPL3, so distributing script from `http://software.virtualmin.com/gpl/scripts/install.sh` is under same conditions (or any compatible copyleft). I making bold move to keep this repository fresh, all contributions authorship reserved to their original committers.

### CODE REVIEW

**Comment by virtualmin** [comment]
This is a regression. We have been sloppy in how we update the script on the website; all three of us have access to the website, and on a couple of occasions there have been changes there that didn't get merged back into the "real" source which is what is in git here.

But, most of the changes in this PR are regressions; moving the script backward, rather than forward. It's my fault for not syncing them up, after the last time Eric made some changes due to a bug.

Your other later changes are good...but, they should ideally be made against what we have in git rather than what is on software.virtualmin.com. I need to spend some time automating deployment of this again; before we moved to git, and things were in SVN, we had automated scripts to pull in the latest version from revision control, but things got sloppy during a bunch of recent migrations.

In short: Don't use the version on software.virtualmin.com; it is not the latest, or most correct, version, even though it looks like it has some new stuff that hasn't been merged in here. I'm not going to merge this PR, but I would be happy to merge your other PR if it is made against the source here rather than the one on software.virtualmin.com.

**Comment by virtualmin** [comment]
And when I say "don't use it", I mean, don't use it to build PR requests. It is fine for installing Virtualmin (though I believe this version in git is slightly more resistant to user error, and it also gets rid of a bunch of old/dead code for distros and package managers that don't exist anymore).

**Comment by Enelar** [comment]
K, when i spare little off time again, i will fix it. So current master is latest, correct?

**Comment by virtualmin** [comment]
Yes, current master is up to date with what is on my dev machine.

- Joe

**Comment by swelljoe** [comment]
I'm gonna close this one, as the rebuild of the install script has been mostly completed. Fedora 25 is not yet supported, but most of what the install script needs is in place, and there are yum groups for it. All that remains is repository support and testing. I believe it's a day or so of work to complete fedora support, but it'll have to come after I wrap up the Virtuamlin 6 rollout.
